specification/deviceupdate/data-plane/readme.python.md

```diff
@@ -4,14 +4,21 @@ These settings apply only when `--python` is specified on the command line.
 Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
 
 ```yaml
-  azure-arm: false
-  license-header: MICROSOFT_MIT_SMALL_NO_CODEGEN
-  namespace: azure.deviceupdate
-  package-name: azure-deviceupdate
-  package-version: 2020-09-01-preview
+  python-mode: create
```

Comment:
by default, you will be creating a whole new generated folder. For subsequent generations once, you can add `--python-mode=update` to your command line to avoid generating everything again